src/api/HelloWorld.ts

```diff
@@ -0,0 +1,33 @@
+/* Import Dependencies */
+import axios from 'axios';
+
+
+/**
+ * Function that checks if the API is online with a 'hello world' protocol
```

Comment:
To display the 'API service is down' message in the header when the API is down. Could also integrate it into other requests, but since the API provides one I thought lets do that for now

Reply:
I removed the Hello World call and instead implemented the error handling into the default requests, updating a global error variable after requesting (200 is true, any error is false).
A bit quick, for DiSSCover I want to do this more thoroughly.